src/TrendIndicator/TrendIndicator.st.css

```
@@ -0,0 +1,45 @@
+:import {
+  -st-from: 'wix-ui-core/hocs.st.css';
+  -st-named: Focusable;
+}
+
+:import {
+  -st-from: "../Foundation/stylable/colors.st.css";
+  -st-named: D40, G10, R10;
+}
+
+.root {
+  -st-extends: Focusable;
+  -st-states: skin(string);
```

We prefer to use explicit values like so:

```suggestion
  -st-states: skin(enum(neutral, positive, negative));
```

---

fixed